core/src/test/scala/org/apache/spark/deploy/security/ConfigurableCredentialManagerSuite.scala

```diff
@@ -64,36 +57,22 @@ class ConfigurableCredentialManagerSuite extends SparkFunSuite with Matchers wit
 
   test("using deprecated configurations") {
     sparkConf.set("spark.yarn.security.tokens.hadoopfs.enabled", "false")
-    sparkConf.set("spark.yarn.security.tokens.hive.enabled", "false")
+    sparkConf.set("spark.yarn.security.credentials.hive.enabled", "false")
     credentialManager = new ConfigurableCredentialManager(sparkConf, hadoopConf)
 
     credentialManager.getServiceCredentialProvider("hadoopfs") should be (None)
     credentialManager.getServiceCredentialProvider("hive") should be (None)
-    credentialManager.getServiceCredentialProvider("test") should not be (None)
     credentialManager.getServiceCredentialProvider("hbase") should not be (None)
   }
 
-  test("verify obtaining credentials from provider") {
+  test("verify no credentials are obtained") {
     credentialManager = new ConfigurableCredentialManager(sparkConf, hadoopConf)
     val creds = new Credentials()
 
-    // Tokens can only be obtained from TestTokenProvider, for hdfs, hbase and hive tokens cannot
-    // be obtained.
+    // Tokens cannot be obtained from HDFS, Hive, HBase in unit tests.
     credentialManager.obtainCredentials(hadoopConf, creds)
     val tokens = creds.getAllTokens
-    tokens.size() should be (1)
-    tokens.iterator().next().getService should be (new Text("test"))
-  }
-
-  test("verify getting credential renewal info") {
```

Comment:
ConfigurableCredentialManager no longer supports service loading, and the service loaded `TestCredentialProvider` was the only provider that returned a renewal time.